src/main/java/teammates/ui/webapi/output/SessionResultsData.java

```diff
@@ -337,6 +337,13 @@ private static String getRecipientNameOfResponse(FeedbackResponseAttributes resp
             this.questionStatistics = questionStatistics;
         }
 
+        protected QuestionOutput(FeedbackQuestionAttributes feedbackQuestionAttributes, String questionStatistics,
```

Comment:
If you are not using `questionStatistics` why define it here in the first place?